src/test/java/de/tonypsilon/bmm/backend/division/service/DivisionServiceTest.java

```diff
@@ -63,33 +64,34 @@ void testGetAllDivisionsOfSeasonByLevel() {
         when(divisionRepository.findBySeasonId(1L)).thenReturn(List.of(
            landesliga, stadtligaA, stadtLigaB));
         SortedSetMultimap<Integer, DivisionData> actual = divisionService.getAllDivisionsOfSeasonByLevel(1L);
-        assertEquals(3, actual.size());
+        assertThat(actual.size()).isEqualTo(3);
```

Comment:
Hi @ujwalkumar1995, here is another .hasSize possible

Reply:
Hey @Tonypsilon 
I tried using hasSize here, but in case of this SortedSetMultimap, it was giving me syntax error. May be it is not supported for SortedSetMultimap. Not sure what else could be the reason here.

Reply:
Hi @ujwalkumar1995, I found out the cause: assertJ for Guava was not yet in the pom.xml. I updated the pom.xml file, feel free to take a pull and try again, .hasSize should work now for MultiMap :)

Reply:
Hey @Tonypsilon 
Thank you for that. I rebased your commit on my branch but I am still getting the syntax error on below line.
![Screenshot_20230303_092347](https://user-images.githubusercontent.com/20976813/222766110-de34a6a3-9cb2-4f45-80fd-f7916a4d7b38.png)

Reply:
Make sure to `import static org.assertj.guava.api.Assertions.*;`
![image](https://user-images.githubusercontent.com/70838192/222773029-7e8810f2-f63a-41f7-bab0-6019e91b2228.png)